xc7/additional_examples/button_controller/Makefile

```diff
@@ -0,0 +1,41 @@
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
hmm, this Makefile is copied in every example directory. Maybe it's about time to extract a common part of it and reuse between the examples?

Reply:
@kgugala Are you suggesting to add a common Makefile to symbiflow and then have each design's Makefile call it? Just to make sure I understand what you are proposing.

Reply:
@kgugala @WhiteNinjaZ To answer Karol's question, maybe I can jump in here as well...  We have been talking with Tim and he would like us to eventually add a fair number of example designs to symbiflow-examples.  They may come from fpgacpu.ca, projectf.io, and some other places.  Our thought was to organize them into directories to keep things manageable.

Reply:
I think that it might be a good idea to have a separate PR that moves the common Makefile in a common folder. This allows us to avoid a lot of code duplication that will come with the addition of code.

The common Makefile can than be included by the various examples, which will then assign the correct values to the different variables (e.g. DEVICE, BITSTREAM_DEVICE, etc.).

Reply:
Just submitted [#174](https://github.com/SymbiFlow/symbiflow-examples/pull/174)